src/lsst/cm/tools/core/panda_utils.py

```diff
@@ -383,7 +383,7 @@ def get_panda_errors(
     errors_aggregate = dict()
     has_merging = False
     for task in tasks:
-        if task["taskname"].find("mergeExecutionButler") >= 0:
+        if task["taskname"].find("xecutionButler") >= 0:
```

Comment:
Do we know why taskname is getting set to xecutionButler? That feels like it might be a PanDA side issue.

Reply:
Resolved over slack DM